src/pseudopeople/loader.py

```diff
@@ -0,0 +1,114 @@
+from pathlib import Path
```

Comment:
Moving these here for multiple reasons:
-avoiding circular dependencies when storing utilies
-This is likely the start of a large refactor we will do later. See MIC-4244